Harvest.Web/ClientApp/src/types.ts

```diff
@@ -174,7 +180,7 @@ export class WorkItemImpl implements WorkItem {
   type;
   rate;
   rateId = 0;
-  unit = 'hr';
+  unit = "hr";
```

Comment:
@srkirkland Do we have or need a prettier file?

Reply:
I think it was prettier that formatted this